README.md

```diff
@@ -83,15 +107,9 @@ Run an [S3 ninja] Docker container, specifying a local directory in which to sav
 docker run -d -p 9444:9000 -v <some directory path>:/home/sirius/data --name=s3ninja scireum/s3-ninja:6.4
 ```
 
-Set the `S3_ENDPOINT_OVERRIDE` environment variable to the upload URL of the emulator, in this case
```

Comment:
These environment variables are no longer necessary because they're set in the new config file.